.github/workflows/release.yml

```diff
@@ -71,7 +64,6 @@ jobs:
   publish-tag:
     name: Publish tag
     needs: build-master
-    if: contains(needs.check-permission.outputs.hasWritePermission, 'true')
```

Comment:
Shoudl this line for the `publish-tag` step be `if: contains(inputs.debug, 'false')` similar to the `publish-docker-image` and `publish-release` steps? When I go to run the release, the checkbox says: "executes the workflow in debug mode (skip the publishing tag, docker image and release steps)".